src/amo/pages/Home/index.js

```diff
@@ -53,17 +54,23 @@ export const isFeaturedCollection = (
 export const getFeaturedCollectionsMetadata = (i18n) => {
   return [
     {
-      footerText: i18n.gettext('See more news boosters'),
-      header: i18n.gettext('News boosters'),
+      footerText: i18n.gettext('See more YouTube boosters'),
+      header: i18n.gettext('YouTube boosters'),
       isTheme: false,
       ...FEATURED_COLLECTIONS[0],
     },
     {
-      footerText: i18n.gettext('See more weather reports'),
-      header: i18n.gettext('Weather reports'),
+      footerText: i18n.gettext('See more health & wellness add-ons'),
```

Comment:
nit: just wondering...should this say "See more health & wellness **extensions**?

Reply:
Yes, it should say "extensions".

Reply:
:+1: